.github/workflows/run-tests.yml

```
@@ -0,0 +1,43 @@
+      test_file:
+        description: 'Comma-separated list of test files to run (e.g., Button.test.tsx,Input.test.tsx).'
+        default: ''
+        required: false
+        type: string
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)